crates/symex_lib/src/any.rs

```diff
@@ -1,46 +1,56 @@
 use super::symbolic;
-use std::mem::MaybeUninit;
 
+/// Provides the [`any`] operation on the implementor
+/// ---
+///
+/// The any operation should return a new [`symbolic`] value with no prior [`assumptions`](crate::assume) placed on the value
```

Comment:
For me at least, it seems misleading to use `` when it doesn't match the functions. Imo using something like "returning an unconstrained value" seems more clear since you can get constraints on value in other ways.

Reply:
That sounds better, will refactor